### DISCUSSION TITLE
Support the WebStoreTemplate metadata type

### DISCUSSION THREAD

**Discussion by marcoalmodova** [scolladon/sfdx-git-delta#774] [Ideas]
Hi, I was checking the files with the API versions and I could not find it there, but can the WebStoreTemplate metadata type be supported? 
We are using the plugin in a project and it does not detect the changes to this type of metadata.

There is the SF doc about it: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_webstoretemplate.htm

Thanks!

**Comment by scolladon** [comment]
Hi @marcoalmodova ! 

Yes sure, I've converted this discussion into an issue to track it there
